lib/commands/config/read.ts

```diff
@@ -61,7 +59,7 @@ export default class ConfigReadCmd extends Command {
 		await safeUmount(drive);
 
 		const config = await import('balena-config-json');
-		const configJSON = await config.read(drive, options.type);
+		const configJSON = await config.read(drive, '');
```

Comment:
This commit, `config inject/read/write: Avoid need for internet connection, ignore device type`, depends on balena-io-modules/balena-config-json/pull/21 
When I suggested cherry picking PR #2369, I mean only commit [044fd5b Add request timeout (5s) for deprecation check and event tracking](https://github.com/balena-io/balena-cli/pull/2369/commits/044fd5b683f25d71dbb3981c96027bc89a94c821)
The rest wasn't ready yet.

Reply:
@pdcastro Right, gotcha, thanks.